frontend/packages/dev-console/integration-tests/support/pageObjects/quickStarts-po.ts

```diff
@@ -0,0 +1,20 @@
+export const quickStartsPO = {
+  quickStartTitle: '[data-test="QuickStartCatalogPageTitle"]',
+  filterKeyword: 'input.pf-c-search-input__text-input',
+  statusFilter: 'button.pf-c-select__toggle',
+  statusDropdown: '[aria-label="Select filter"]',
+  statusComplete: '[data-key="Complete"]',
+  installOpenshiftPipelineOperatorCard:
+    '[data-test="tile-Install the OpenShift Pipelines Operator"]',
+  getStartedWithASampleApplicationCard: '[data-test="tile-Get started with a sample application"]',
+  emptyState: 'div.pf-c-empty-state__content',
+  clearFilter: '[data-test="clear-filter-button"]',
+  cardStatus: '[data-test="quick-start-card-status"]',
+};
+
+export const quickStartSidebarPO = {
+  quickStartSidebarBody: '[data-test="quickstart-sidebar-content"]',
+  startButton: '[data-test="Start-quick-starts"]',
+  nextButton: '[data-test="Next-quick-starts"]',
+  closeButton: '[data-test="Close-quick-starts"]',
```

Comment:
```suggestion
  quickStartSidebarBody: '[data-test~="drawer"] [data-test~="content"]',
  startButton: '[data-test~="drawer"] [data-test="Start button"]',
  nextButton: '[data-test~="drawer"] [data-test="Next button"]',
  closeButton: '[data-test~="drawer"] [data-test="Close button"]',
```

Reply:
Instead of doing `[data-test~="drawer"] [data-test="Start button"]` for the button I've written it as `[data-test="Start button"]` in this file and in the code I used the `quickStartSidebarBody` page object and find the button element(startButton) in it as it is doing similar work and not hindering my test result